packages/kbn-es/src/ess_resources/service_tokens

```diff
@@ -1 +1,2 @@
-elastic/kibana/kibana-dev:$2a$10$mY2RuGROhk56vLNh.Mgwue98BnkdQPlTR.yGh38ao5jhPJobvuBCq
\ No newline at end of file
+elastic/kibana/kibana-dev:$2a$10$mY2RuGROhk56vLNh.Mgwue98BnkdQPlTR.yGh38ao5jhPJobvuBCq
+elastic/fleet-server/fleet-server-dev:$2a$10$tgMX7U09G/EVTP8F/O4zHewhA3DXdv7iM5F2vny9TC6zw77RrutyG
```

Comment:
Mind adding the encoded value of this token to `packages/kbn-es/src/ess_resources/README`? Maybe worth making a constant as well for future usage. For `kibana-dev` token we put it in `packages/kbn-dev-utils/src/dev_service_account.ts`

Reply:
Hey, sure, I'll add it to readme :+1: , however, since I am not using this token anywhere directly, I am not sure if it makes sense to create a const, Should I create a unused one?

Reply:
Probably only the readme is necessary then, thanks!